app/src/main/java/org/mozilla/fenix/downloads/DownloadsItemMenu.kt

```diff
@@ -0,0 +1,30 @@
+package org.mozilla.fenix.downloads
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
nit: the license header should be at the very top of the file, above the package statement - https://developer.android.com/kotlin/style-guide#structure